haskell-tests/Generators.hs

```diff
@@ -297,6 +297,7 @@ genPayloadConfigureBaker = do
     cbTransactionFeeCommission <- liftArbitrary genAmountFraction
     cbBakingRewardCommission <- liftArbitrary genAmountFraction
     cbFinalizationRewardCommission <- liftArbitrary genAmountFraction
+    cbSuspend <- if pv < P8 then return Nothing else arbitrary
```

Comment:
As a general rule, I'd suggest preferring to test `supportsAccountSuspension (accountVersionFor pv)` which is clearer semantically.

Reply:
done